### PR TITLE
Improve dashboard documentation (part 3)

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

Follow-up to #11065. This PR adds the following docs:

- **Import, export, and print a snapshot**
- **Customize the standard dashboard**

The last push on this front will be the new web server reference, which is mostly done!

### CODE REVIEW

**Comment by joelhans** [comment]
@OdysLam Ping!

**Review by OdysLam — APPROVED** [review]

File: docs/dashboard/customize.mdx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+Find this setting under the **Visual** tab. By default, Netdata places the [legend of
+dimensions](/docs/dashboards/charts-dimensions-contexts-families.mdx#dimensions) _below_ charts. Click this toggle to
+move the legend to the _right_ of charts.
+
+### Change theme
+
+Find this setting under the **Visual** tab. Choose between Dark (the default) and White.
+
+## Customize the standard dashboard
```
Comment (ilyam8) [inline docs/dashboard/customize.mdx:35]: [we have the same section in the web/gui](https://github.com/netdata/netdata/tree/master/web/gui#customizing-the-local-dashboard)

Do you plan to remove it?

> The last push on this front will be the new web server reference, which is mostly done!

I read the comment, but I am not sure you are referencing to the web/gui.

**Review by ilyam8 — APPROVED** [review]

File: docs/dashboard/customize.mdx (hunk 1 above)
Comment (joelhans) [reply docs/dashboard/customize.mdx:35]: Yes, the next PR will add a new reference file and then remove a number of files that are now duplicates, like `/web/gui/README.md`.

Comment (ilyam8) [reply docs/dashboard/customize.mdx:35]: Thanks for the clarification!

**Pull request merged**